types/handsontable/index.d.ts

```diff
@@ -502,18 +503,10 @@ declare var Handsontable: {
     utils: ht.Utils;
     renderers: ht.Renderers;
     editors: ht.Editors;
-};
+}
 
 declare module "handsontable" {
```

Comment:
@sandersn, thanks for the feedback! I made the requested changes. This worked in one project but not in my this or my test project. Can you help me understand/diagnose what's wrong?